spec/filters/geoip_spec.rb

```diff
@@ -1,12 +1,35 @@
 require "logstash/devutils/rspec/spec_helper"
 require "logstash/filters/geoip"
 
+ASNDB = ::Dir.glob(::File.expand_path("../../vendor/", ::File.dirname(__FILE__))+"/GeoIPASNum*.dat").first
```

Comment:
when I tried it did not work and I did not feel like looking for the reason since this is an "old" style specs that desperately need to be refactored anyway.